MapboxMobileEvents/MMECommonEventData.m

```diff
@@ -71,16 +72,22 @@ - (instancetype)init {
 
 + (NSString *)applicationState {
 #if TARGET_OS_IOS || TARGET_OS_TVOS
-    switch (UIApplication.sharedApplication.applicationState) {
-        case UIApplicationStateActive:
-            return MMEApplicationStateForeground;
-        case UIApplicationStateInactive:
-            return MMEApplicationStateInactive;
-        case UIApplicationStateBackground:
-            return MMEApplicationStateBackground;
-        default:
-            return MMEApplicationStateUnknown;
+
+    if (NSBundle.mme_isExtension) {
+        return MMEApplicationStateUnknown;
```

Comment:
I wasn't quite sure how to port these values in the event of running in an extension. Extensions offer a much more limiting running time being killed immediately after dismissal.

Options:
- Return active by default if in extension?
- Return Unknown given there isn't a value to check?
- Other?

Reply:
Alternative:
Drop key/value all together if extension? Or the full event if extension?

Reply:
Let's add another value for `MMEApplicationStateExtension` and merge. We discussed this in a recent meeting and think it makes sense to let this fly with a known value to observe.